npm-debug.log

```diff
@@ -0,0 +1,94 @@
+0 info it worked if it ends with ok
```

Comment:
you should add .npm-debug.log to the `.gitignore` file, it should not be commited